package.json

```diff
@@ -36,58 +36,58 @@
     "dom-helpers": "^3.2.1",
     "prop-types": "^15.5.10",
     "prop-types-extra": "^1.0.1",
-    "react-transition-group": "^2.0.0-beta.0",
+    "react-transition-group": "^2.2.0",
     "warning": "^3.0.0"
   },
   "peerDependencies": {
     "react": "^0.14.9 || >=15.3.0",
     "react-dom": "^0.14.9 || >=15.3.0"
   },
   "devDependencies": {
-    "@monastic.panic/component-playground": "^2.0.0",
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.25.0",
-    "babel-eslint": "^7.2.3",
-    "babel-loader": "^7.0.0",
+    "@monastic.panic/component-playground": "^3.1.0",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "^8.0.0",
```

Comment:
not sure about using 8.0.0 here – that one relies on babel 7, which we're not otherwise using

Reply:
I'll change it back then.